src/main/java/teammates/ui/controller/InstructorCourseEnrollResultPageData.java

```diff
@@ -8,27 +9,69 @@
 import teammates.common.datatransfer.StudentAttributes.UpdateStatus;
 import teammates.common.util.Const;
 import teammates.common.util.Utils;
+import teammates.ui.template.EnrollResultPanel;
 
 /**
  * PageData: page data for the 'Result' page after enrollment for a course
  */
 public class InstructorCourseEnrollResultPageData extends PageData {
+    protected static final Logger log = Utils.getLogger();
+    
+    private String courseId;
+    private List<StudentAttributes>[] students;
+    private boolean hasSection;
+    private String enrollStudents;
+    private List<EnrollResultPanel> enrollResultPanelList;
     
-    public InstructorCourseEnrollResultPageData(AccountAttributes account) {
+    public InstructorCourseEnrollResultPageData(AccountAttributes account, String courseId, 
+                                                List<StudentAttributes>[] students, boolean hasSection, 
+                                                String enrollStudents) {
         super(account);
+        this.courseId = courseId;
+        this.students = students;
+        this.hasSection = hasSection;
+        this.enrollStudents = enrollStudents;
+        enrollResultPanelList = new ArrayList<EnrollResultPanel>();
+        
+        for (int i = 0; i < UpdateStatus.STATUS_COUNT; i++) {
+            String panelClass = "";
+            if (i == UpdateStatus.ERROR.numericRepresentation) {
+                panelClass = "panel panel-danger";
+            } else if (i == UpdateStatus.NEW.numericRepresentation) {
+                panelClass = "panel panel-primary";
+            } else if (i == UpdateStatus.MODIFIED.numericRepresentation) {
+                panelClass = "panel panel-warning";
+            } else if (i == UpdateStatus.UNMODIFIED.numericRepresentation) {
+                panelClass = "panel panel-info";
```

Comment:
Extracting this into another method like `getMessageForEnrollmentStatus()` with a switch-case should make this easier to read.
Also, considering they all start with "panel ", why not just put that on the frontend instead?